app/javascript/src/components/product-card.vue

```diff
@@ -101,6 +116,11 @@ import priceToSigns from '../utils/price-to-signs';
 const CLICKS_BEFORE_NOTIFICATION = 10;
 
 export default {
+  data() {
+    return {
+      whatsappNumber: process.env.PHONE_NUMBER,
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'process' is not defined  no-undef